config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml

```diff
@@ -11,7 +11,7 @@ presubmits:
       description: "Verify source code of node-feature-discovery-operator: check formatting, lint etc."
     spec:
       containers:
-      - image: golang:1.18
+      - image: golang:1.19
```

Comment:
sorry to nit about this but the commit message title is not aligned with this 🙈

Maybe just `Bump go version in NFD test jobs` or smth?